.github/workflows/ql-for-ql-build.yml

```
@@ -146,12 +166,26 @@ jobs:
 
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Create CodeQL config file
+        run: |
+          echo "paths:" > ${CONF}
+          echo "  - ${{ matrix.folder }}" >> ${CONF}
```

```suggestion
          echo "  - ${FOLDER}" >> ${CONF}
```
+ env change below

(this replaces dynamically generated bash code with simple variable substitution which is much safer in general)